### DISCUSSION TITLE
Can this lib be used in Nuxt 3?

### DISCUSSION THREAD

**Discussion by rowild** [apertureless/vue-chartjs#1032] [Q&A]
I tried this:

```js
// plugins/charts.js

import { Bar } from "vue-chartjs"

export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.vueApp.use(Bar)
})
```

But I get a whole list of warnings:

```sh
chunk-LGAZO65B.js:1381 [Vue warn]: A plugin must either be a function or an object with an "install" function.
warn2 @ chunk-LGAZO65B.js:1381
use @ chunk-LGAZO65B.js:5153
(anonymous) @ charts.client.js:6
wrapper @ nuxt.js:189
...
chunk-LGAZO65B.js:1381 [Vue warn]: Failed to resolve component: BarChart
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement. 
  at <Chart onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< undefined > > 
  at <RouteProvider key="/health-topics/blood-pressure/chart" routeProps= {Component: {…}, route: {…}} pageKey="/health-topics/blood-pressure/chart"  ... > 
...
```

What do I have to do to get this right?

**Comment by CareTiger** [comment]
@rowild  I am interested in using this as well in my nuxt 3 project. sadly documentation is lacking

**Comment by rowild** [comment]
@CareTiger Have a look at this post here:
https://stackoverflow.com/questions/76236369/what-is-the-proper-way-to-include-chart-js-with-nuxt-3-4/76240253#76240253

I could not really dig into it yet, since I am currently not at my computer, but the stackblitz example seems to work fine.

**Reply by rowild** [reply]
@CareTiger Thank you for the link! Very good resource!

(Sorry for the late response, I am only seeing this today.)

**Reply by Suniron** [reply]
I found this solution harder to maintain since we have to find each missing import to have all parts of a Chart 🥲.

The solution to import all elements works but it's not optimized ..
